figures/compat.py

```diff
@@ -26,12 +26,13 @@
         # try the pre-ginkgo path
         from lms.djangoapps.grades.new.course_grade import CourseGradeFactory    # noqa: F401
 
-try:
```

Comment:
@OmarIthawi  Why not just do this? (and not add the extra function)

```
if RELEASE_LINE == 'ginkgo':
    from certificates.models import GeneratedCertificate
else:
    from lms.djangoapps.certificates.models import GeneratedCertificate
```

Also I seem to remember a PR review comment you made to me that I should avoid using getter methods when we can just import the object directly in the flavor of 'Six'

Reply:
Sure thing @johnbaldwin! Done.